### DISCUSSION TITLE
Not able to edit the space setting on demo.snapshot

### DISCUSSION THREAD

**Discussion by katariachetan** [snapshot-labs/snapshot#3308] [Q&A]
I am not able to add author or edit any settings on my space named OrangePool. Attached the document. Is there any time validity for existence for space on demo.snapshot site?

![image](https://user-images.githubusercontent.com/117351209/199741763-b3ccbda7-2313-4d3c-9e93-b9ee147f0d6a.png)

**Comment by metrox-eth** [comment]
Hey, you've probably created your test space with a Rinkeby ENS before the switch to Goerli. A dump of the data is available upon request.

**Reply by katariachetan** [reply]
I have created space with Goerli network as Rinkeby ENS is depreciated for metamask

**Reply by metrox-eth** [reply]
What's your ENS ?

**Reply by metrox-eth** [reply]
Hm I can't reproduce
<img width="1076" alt="Capture d’écran 2022-11-03 à 21 15 48" src="https://user-images.githubusercontent.com/55286013/199744676-76a2a4f7-1ac7-45b1-95e4-22c7fcada71b.png">

**Reply by katariachetan** [reply]
Please check here. Attached the screenshot
![image](https://user-images.githubusercontent.com/117351209/199770949-0addb256-e939-46d9-aeb9-352a6e3dd406.png)

**Reply by katariachetan** [reply]
Even I am not my ens option while creating a space again. Could you please share the reason. Attached the screenshot 
![image](https://user-images.githubusercontent.com/117351209/199774315-4e3fdfa4-8320-4400-8d90-45cd2b0102f0.png)

**Reply by metrox-eth** [reply]
By looking at your screenshot I see that your wallet is not properly connected, something is blocking, can you check if you have an adblocker active ?

**Comment by katariachetan** [comment]
yes you are right Adblocker active on chrome & have disabled it. Now I can see the settings options for space. Attached the screenshot.
![image](https://user-images.githubusercontent.com/117351209/199899690-70eb5794-73cc-4683-8dcd-37350237c0f8.png)
But when I am trying to create a new space till I am not seeing my old ENS names. Can you please help me with this? Attached the screenshot.
![image](https://user-images.githubusercontent.com/117351209/199899924-33fb8e4d-894d-4c9c-82a7-7bc10ef09dcb.png)

**Reply by metrox-eth** [reply]
Your space orangepool.eth is already created. With which ENS do you want to create an other space ? You can't create more than one space per ENS, but you can use a subdomain i.e. newspace.orangepool.eth, you'll need to create this subdomain on ENS app.

**Reply by katariachetan** [reply]
Thank you for sharing the information. It was really helpful. I am having 2 ENS app - impossibledao.eth & orangepool.eth. I have created a new space in orangepool.eth but I want to create another space in impossibledao.eth. Just wanted to check why it is not been displayed. 

![image](https://user-images.githubusercontent.com/117351209/199923878-24544f00-4d0a-4549-87dc-c5d96abb51da.png)

**Reply by metrox-eth** [reply]
This is because you've already created this space https://demo.snapshot.org/#/impossibledao.eth

**Reply by katariachetan** [reply]
Thank you. It was really helpful for you. You replied so fast. Really appreciate it. One last thing I want to ask from you. How did you find that a particular ENS app there is a associated space created. Could you please share the process with me?

**Reply by metrox-eth** [reply]
You can add the ENS name to snapshot URL https://demo.snapshot.org/#/ensname.eth or check on the ENS app if there is a snapshot text text record.

**Reply by metrox-eth** [reply]
Or you can type the ENS in snapshot search field, it's even easier.

**Reply by katariachetan** [reply]
Thank you so much for your help & support
